src/extension.ts

```diff
@@ -121,20 +70,61 @@ function decorateEditor(editor: sourcegraph.CodeEditor, sentryProjectId: string,
 }
 
 /**
- * Build URL to the Sentry issues stream page with the Sentry Org, Sentry project ID, and query.
+ * Decorate a line that matches either the line match pattern from the Sentry extension configurations
+ * or that matches common error loggin patterns.
+ * @param i index for decoration range
+ * @param m line match containing error query
+ * @param sentryProjectId Sentry project id retrieved from Sentry extension settings
+ */
+export function decorateLine(
+    i: number,
+    m: RegExpExecArray,
+    sentryProjectId?: string
+): sourcegraph.TextDocumentDecoration {
+    let decoration: sourcegraph.TextDocumentDecoration
+    decoration = {
+        range: new sourcegraph.Range(i, 0, i, 0),
+        isWholeLine: true,
+        after: {
+            backgroundColor: '#e03e2f',
+            color: 'rgba(255, 255, 255, 0.8)',
+            contentText: !SENTRYORGANIZATION
+                ? ' Add Sentry extension configurations to your settings to view logs in Sentry » '
```

Comment:
I like the idea of "hinting" decorations when the extension isn't configured, but the wording and the `»` here would make me think that these decorations are actionable, while they aren't. Two options:
- Remove `»`, add a more helpful hover message (this should only be a temporary fix)
- This link is actually active, and links to `$SOURCEGRAPH_URL/users/$USER/settings`. You could get the current username through graphQL.

Also: this text is quite long, which means it can end up being cropped quite easily. I suggest:

`Configure the Sentry extension to view logs` (33 chars shorter)

Reply:
Yes, this will be addressed in a separate PR.